adr/recipes/files_with_coverage.py

```diff
@@ -22,7 +22,13 @@ def run(args, config):
     header = ['Revision', 'Files With Coverage', 'Total Files', 'Percent with Coverage']
     covered_files = next(run_query('covered_files', config))['data']
     total_files = next(run_query('total_files', config))['data']
-
+    
+    if None in  [item for items in covered_files for item in items]:
+        raise MissingDataError("ActiveData returned null value.")
```

Comment:
Please move this above the call to `total_files`. That way if data is missing here, we won't run the second query unnecessarily.